dash.js

```diff
@@ -179,12 +179,15 @@ var MyDash = new Lang.Class({
     Name: 'DashToDock.MyDash',
 
     _init: function(settings, remoteModel, monitorIndex) {
+        this._dtdSettings = settings;
+
+        // Initialize icon variables and size
         this._maxHeight = -1;
-        this.iconSize = 64;
+        this.iconSize = settings.get_int('dash-max-icon-size');
```

Comment:
to avoid confusion with the call below, use here this._dtdSettings as well?

Reply:
Done!